worlds/adventure/docs/en_Adventure.md

```diff
@@ -1,11 +1,11 @@
 # Adventure
 
-## Where is the settings page?
-The [player settings page for Adventure](../player-settings) contains all the options you need to configure and export a config file.
+## Where is the options page?
+The [player options page for Adventure](../player-options) contains all the options you need to configure and export a config file.
 
 ## What does randomization do to this game?
 Adventure items may be distributed into additional locations not possible in the vanilla Adventure randomizer.  All
-Adventure items are added to the multiworld item pool.  Depending on the settings, dragon locations may be randomized,
+Adventure items are added to the multiworld item pool.  Depending on the options, dragon locations may be randomized,
```

Comment:
```suggestion
Adventure items are added to the multiworld item pool.  Depending on the `dragon_rando_type` value, dragon locations may be randomized,
```

Reply:
Looks good.  No reason not to specify it unless additional options interfere it, and I don't think they do.